net_infrastructure/comware_5_2_port.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/python
+#coding: utf-8 -*-
+
+# (c) 2014, Patrick Galbraith <patg@patg.net>
+#
+# This file is part of Ansible
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: comware_5_2_port
+version_added: 0.1
+author: Patrick Galbraith
+short_description: Manage Ports on Comware 5.x Switches
+requirements: [ paramiko comware_5_2 (http://code.patg.net/comware_5_2.tar.gz)]
+description:
+    - Manage Ports on Comware 5.2-based Switches
+options:
+    developer-mode:
+        required: false
+        default: true
+        choices: [ true, false ]
+        description:
+            - Whether to set the switch into developer mode. Switch doesn't
+              much when not in developer mode!
+    save:
+        required: false
+        default: false
+        choices: [ false, true ]
+        description:
+            - if true, all changes will be written. Upon reboot, save
+    state:
```

Comment:
matching states would be desired (up/down, enabled/disabled, on/off, started/stopped)